etc/docker/kernel-tf-py/Dockerfile

```diff
@@ -1,5 +1,6 @@
 # Ubuntu:Bionic
-ARG BASE_CONTAINER=jupyter/tensorflow-notebook:04f7f60d34a6
+ARG BASE_CONTAINER=jupyter/tensorflow-notebook:703d8b2dcb88
```

Comment:
Should we add a comment that indicates the tensorflow version here?  That might be helpful to maintainers.